droidlet/dashboard/web/src/components/MobileDirectionButton.js

```diff
@@ -20,13 +36,47 @@ class MobileDirectionButton extends React.Component {
     });
   }
 
+  /**
+   * sends commands to stateManager
+   * clears commands
+   */
+  sendAndClearCommands() {
+    if (this.state) {
+      stateManager.buttonHandler(this.state.commands);
+      // clears the commands once sent
+      this.setState({
+        commands: [],
+      });
+    }
+  }
+
   render() {
     return (
-      <div>
-        <button onClick={() => this.addCommand("MOVE_LEFT")}>LEFT</button>
-        <button onClick={() => this.addCommand("MOVE_FORWARD")}>UP </button>
-        <button onClick={() => this.addCommand("MOVE_DOWN")}>DOWN </button>
-        <button onClick={() => this.addCommand("MOVE_RIGHT")}>RIGHT</button>
+      <div className="container">
+        <button
+          className="directionButton left"
+          onClick={() => this.addCommand("MOVE_LEFT")}
+        >
+          LEFT
+        </button>
+        <button
+          className="directionButton up"
+          onClick={() => this.addCommand("MOVE_FORWARD")}
+        >
+          UP{" "}
```

Comment:
What's the significance of `{" "}`?

Reply:
Not sure, might've snuck in with the linter. Removed